### PR TITLE
Fix custom signers

### PR DESCRIPTION
**Pull request by jribbink**
Closes #???

## Description

---

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-js-testing/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

**Comment by tav** [comment]
PR is looking good — do you want to set the default `hashAlgorithm` to `SHA3_256` as it was before by any chance?

**Comment by tav** [comment]
Btw, the `@onflow/flow-js-testing` package on npm doesn't seem to have the `dist` directory, so you can't actually import it 😞

**Comment by jribbink** [comment]
> Btw, the `@onflow/flow-js-testing` package on npm doesn't seem to have the `dist` directory, so you can't actually import it 😞

Hmmm it should.  If I pull it up on unpkg I can see the dist folder https://unpkg.com/browse/@onflow/flow-js-testing@0.3.0-alpha.13/

File: dev-test/account.test.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  HashAlgorithm,
+  SignatureAlgorithm,
+} from "../src"
+import {playgroundImport} from "../src/transformers"
+import * as fcl from "@onflow/fcl"
+import {ec as EC} from "elliptic"
+import {validateKeyPair} from "./util/validate-key-pair"
+
+// We need to set timeout for a higher number, because some transactions might take up some time
+jest.setTimeout(10000)
```
Comment (chasefleming) [inline dev-test/account.test.js:28]: Which transactions is this waiting on?

File: dev-test/account.test.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+  expect(addr1).toBe(addr2)
+})
+
+it("createAccount - should work without name and returns address", async () => {
+  const Billy = await createAccount({
+    name: "Billy",
+  })
+
+  expect(Billy).toMatch(/^0x[0-9a-f]{16}$/)
```
Comment (chasefleming) [inline dev-test/account.test.js:55]: Would `/^0x[0-9a-f]{16}$/` be nice as a helper? I see it reused a few times and people might like a helper

File: dev-test/account.test.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      },
+    ],
+  })
+
+  expect(Billy).toMatch(/^0x[0-9a-f]{16}$/)
+
+  const keys = await fcl.account(Billy).then(d => d.keys)
+
+  expect(keys.length).toBe(2)
+  expect(keys[0].hashAlgoString).toBe("SHA2_256")
```
Comment (chasefleming) [inline dev-test/account.test.js:79]: Would be cool to have `SHA2_256` and others as constants you can import to test against.

File: dev-test/account.test.js (hunk 1 above)
Comment (jribbink) [reply dev-test/account.test.js:28]: None, I had just templated the test file from the existing ones and they all had the increased jest timeout.  This is likely unnecessary and I can remove it.

File: dev-test/account.test.js (hunk 2 above)
Comment (jribbink) [reply dev-test/account.test.js:55]: I could make a custom jest matcher for `expect(Foo).toBeAddress()`

File: src/utils.js [hunk 4]
```diff
@@ -19,6 +19,8 @@
 import {createServer} from "net"
 
 export const isObject = arg => typeof arg === "object" && arg !== null
+export const isString = obj => typeof obj === "string" || obj instanceof String
+export const isAddress = address => /^0x[0-9a-f]{0,16}$/.test(address)
```
Comment (chasefleming) [inline src/utils.js:23]: Cool!

File: docs/accounts.md [hunk 5]
```diff
@@ -4,11 +4,25 @@ sidebar_title: Accounts
 description: How to manage accounts addresses
 ---
 
+## Overview
+
 Flow accounts are not derived from a private key. This creates an issues for testing, since
 you need to create actors in a specific order to use their addresses properly.
-In order to reduce this friction we made a handy method `getAccountAddress`, which allows you to access specific address using an alias. This way you can think about actual actors - for example `Alice` and `Bob` - without needing to know their Flow addresses.
+
+In order to reduce this friction we made a handy method, `getAccountAddress`, which allows you to access specific address using an alias. This way you can think about actual actors - for example `Alice` and `Bob` - without needing to know their Flow addresses.
```
Comment (chasefleming) [inline docs/accounts.md:12]:
```suggestion
To reduce this friction, `getAccountAddress`, allows you to access a specific address using an alias. This way you can think about actual actors -- for example `Alice` and `Bob` -- without needing to know their Flow addresses.
```

**Review by chasefleming — APPROVED** [review]
A couple suggestions, but approved.

**Pull request merged**